### ISSUE TITLE
Does not work. Bad tutorial.

### ISSUE THREAD

**Issue by markquinnuk** [dotnet/AspNetCore.Docs#22175]
Hi,

This tutorial does not work. I have followed it to the letter several times and things grind to a halt here.

I get to this point

Test the app
Run the app and append /Movies to the URL in the browser (http://localhost:port/movies).

and get a 404 Page does not exist.

---
#### Document details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6719f08e-3bd7-dc1a-71df-f2ef9fbca9d8
* Version Independent ID: 7096fdb3-612e-9e00-bd0b-8ea4886a09ce
* Content: [Part 2, add a model](https://docs.microsoft.com/en-gb/aspnet/core/tutorials/razor-pages/model?view=aspnetcore-5.0&tabs=visual-studio)
* Content Source: [aspnetcore/tutorials/razor-pages/model.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/tutorials/razor-pages/model.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by Rick-Anderson** [comment]
Hi @markquinnuk 
We have about 50,000 customers per month telling us it does work. Are you using the right port #?

**Comment by markquinnuk** [comment]
Hi,

Visual Studio chooses the port and this is randomly chosen each time the
application is run. I don't think this is the reason for the issue I am
having.

The fact that you say customers confuses me. I am talking about a free
intro tutorial to ASP.NET Core that I am following on a Microsoft site
here Tutorial:
Get started with Razor Pages in ASP.NET Core | Microsoft Docs
<https://docs.microsoft.com/en-gb/aspnet/core/tutorials/razor-pages/razor-pages-start?view=aspnetcore-5.0&tabs=visual-studio>
.

I have followed the guide precisely with a fresh install of Visual Studio.
Three times I have got to the same point and it has not worked. I will get
round to trying a fourth time but I am following another guide here that
actually works https://www.youtube.com/watch?v=BfEjDD8mWYg

When I go to do the scaffolding in your tutorial it fails and I restart VS
and then it works. I think the problem I am having must have something to
do with this. The app has no route for localhost:#####/movies basically. I
think I remember a long time ago there being an issue in VS where you had
to build the project before scaffolding. This other tutorial I am following
builds the CRUD by adding a controller and this works fine.

Thanks,
Mark
